### PR TITLE
adding support for kobo to submit location

### PR DESCRIPTION
**Pull request by tristanhcole**
Kobo can now submit gps location data as name: 'geolocation' type: gps point

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Improve Map Functionality](https://trello.com/c/waznab2T/92-improve-map-functionality)

File: app/server/api/user_api.py [hunk 1]
```diff
@@ -125,7 +126,10 @@ def get(self, user_id):
     def post(self, user_id):
 
         post_data = request.get_json()
-        organisation = g.active_organisation
+        organisation = g.get('active_organisation')
+        if organisation is None:
+            # TODO: Refactor.. Query string needed for kobo
+            organisation = Organisation.query.get(request.args.get('org'))
```
Comment (tristanhcole) [inline app/server/api/user_api.py]: This is a short term fix

File: app/server/utils/user.py [hunk 2]
```diff
@@ -251,7 +256,6 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
         )
 
         transfer_account.name = transfer_account_name
-        transfer_account.location = location
```
Comment (tristanhcole) [inline app/server/utils/user.py]: creating new users with location has been broken for a while it seems

File: app/server/api/user_api.py (hunk 1 above)
Comment (michielderoos) [reply app/server/api/user_api.py]: Will a user ever not have an `active_organisation?

Comment (tristanhcole) [reply app/server/api/user_api.py]: no, but for external auth credentials organisation would need to be pulled from query string (insecure), until your PR is merged in https://github.com/teamsempo/SempoBlockchain/pull/140

Comment (tristanhcole) [reply app/server/api/user_api.py]: could just merge in your PR first ...

**Comment by tristanhcole** [comment]
Merge in PR #140 first

**Pull request merged**